### DISCUSSION TITLE
Where are the file for i18n ?

### DISCUSSION THREAD

**Discussion by Gozer-404** [termux/termux-app#3215] [Q&A] [answered]
Hi guyys,

I downloaded the source code from github (https://github.com/termux/termux-app/releases/tag/v0.118.0) and I'm fighting trying to find the files for the translations.
I'm not really a big android dev expert, but weren't they suppose to live in  app/src/main/res/values-<lang_code> ?

**Comment by agnostic-apollo** [comment, marked as answer]
The `termux-app` only supports english language currently.

**Comment by Gozer-404** [comment]
Hi,
So sorry for the question, it was obvious that termux was not translated
(in fact I was initially searching for the i18n files for MiX explorer and
made a confusion between the two! ).
If you have any interest in getting termux app translated (in French 😁) I
can help you with that.
I've participated in numerous French versions on various platforms (my
'main' one is still for VLC)

Kind regards

Le ven. 27 janv. 2023 à 10:22, agnostic-apollo ***@***.***> a
écrit :

**Reply by agnostic-apollo** [reply]
It may be done in future if needed, but not an immediate priority for now.

Moreover, the termux community is english only since we don't provide support in other languages, so translating app could be problematic for bug reports since our devs may not understand important parts. That is not as much of a problem if external docs are translated to other languages. The site/docs are provided in chinese too and some of our own devs understand it and are willing to keep it upto date.
